### PR TITLE
Refactor manager api and add batch parachains onboarding

### PR DESCRIPTION
**Pull request by PierreBesson**
1. Refactor API endpoints to be more consistent and logical
2. Implement batch parachains on/offboarding in API and UI

Screenshot from 2022-12-29 16-32-08

### CODE REVIEW

File: app/templates/parachains.html [hunk 1]
```diff
@@ -73,9 +72,21 @@ <h3>{{ network_name.capitalize() }} : {{ paras_count }} Paras: {{ parachain_coun
...
+            if (element.checked){
+                paraInputCheckboxes.forEach((paraInput) => paraInput.checked = true)
+            } else {
+                paraInputCheckboxes.forEach((paraInput) => paraInput.checked = false)
+            }
+        }
+
+        window.onboardSelectedParachains = (element) => {
+            paraInputCheckboxes = document.querySelectorAll('input[name="para_id"]');
+            selectedParaIds =  Array.from(paraInputCheckboxes).filter(input => input.checked).map(input => input.value);
```
Comment (BulatSaif) [inline app/templates/parachains.html:86]: if parachain is already onboard will it onboard it again?

**Review by BulatSaif — APPROVED** [review]
The rest is good!

File: app/templates/parachains.html (hunk 1 above)
Comment (PierreBesson) [reply app/templates/parachains.html:86]: Good point, we used to do this check in the front-end. Now this has to be implemented in the back-end.

Comment (PierreBesson) [reply app/templates/parachains.html:86]: Implemented in https://github.com/paritytech/testnet-manager/pull/37/commits/646185821812cc45982e14fa8c39d25cb53d3dcb

**Pull request merged**